app/models/appeals/AppealSubmission.scala

```diff
@@ -67,20 +77,25 @@ object AppealSubmission {
     }
   }
 
-  def constructModelBasedOnReasonableExcuse(reasonableExcuse: String)
+  def constructModelBasedOnReasonableExcuse(reasonableExcuse: String, isLateAppeal: Boolean)
                                            (implicit userRequest: UserRequest[_]): AppealSubmission = {
     reasonableExcuse match {
       case "crime" => {
         AppealSubmission(
-          submittedBy = if(userRequest.isAgent) "agent" else "client",
+          submittedBy = if (userRequest.isAgent) "agent" else "client",
           penaltyId = userRequest.session.get(SessionKeys.penaltyId).get,
           reasonableExcuse = reasonableExcuse,
           honestyDeclaration = userRequest.session.get(SessionKeys.hasConfirmedDeclaration).get == "true",
           appealInformation = CrimeAppealInformation(
             `type` = "crime",
             dateOfEvent = userRequest.session.get(SessionKeys.dateOfCrime).get,
             reportedIssue = userRequest.session.get(SessionKeys.hasCrimeBeenReportedToPolice).get == "yes",
-            statement = None //TODO: may need to add this in later
+            statement = None,
+            lateAppeal = isLateAppeal,
+            lateAppealReason = userRequest.session.get(SessionKeys.lateAppealReason).getOrElse("") match {
```

Comment:
Should be able to replace `userRequest.session.get(SessionKeys.lateAppealReason).getOrElse("")` with `userRequest.session.get(SessionKeys.lateAppealReason)` 

Which would return None if the page hasn't been submitted, or Some("xxx") where the appeal reason has been entered (Some("") if nothing entered but page submitted)?

Reply:
I tried this initially however an empty string is returned from the session (when the user is presented the question but does not answer) therefore in Wireshark I could see the reason was being sent through to the backend with a null value

Reply:
That makes sense, I thought there would be a reason.